tests/integration-tests/tests/common/schedulers_common.py

```diff
@@ -469,6 +469,12 @@ def get_node_info(self, nodename):
         """Get node info."""
         return self._remote_command_executor.run_remote_command("scontrol show nodes {0}".format(nodename))
 
+    def get_conf_param(self, param):
+        """Get value of configuration parameter."""
+        result = self._remote_command_executor.run_remote_command("scontrol show config | grep {0}".format(param))
+        match = re.search(r"(\s+)= (.*)$", result.stdout)
```

Comment:
Are we sure that we only need to parse setting values that do not require any escaping, e.g. `param_name = 'foo bar foo'` we'd like to return the string `foo bar foo` without quotes?

Reply:
As far as I can see, the `scontrol show config` command does not print quotes even if they are present in the configuration files. Also, I manually tested some particular cases (brackets, command, spaces) and it seems to work fine.

But good point, I should add some unit test for this function with an output of this command from a real cluster.

Reply:
@gmarciani and I checked together the format of the output of the command. There's no need to escape quotes.